scripts/build/local_install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
This is the same as klee_deps?

Reply:
Nope, it's for your local installation dependency setup outside of docker - in case you want one.

Reply:
So why is it part of Docker PR?

Reply:
the name of this PR is a bit misleading - I updated it's description